src/ui/components/Library/Team/View/NewTestRuns/RunStats.tsx

```diff
@@ -16,16 +16,25 @@ export function RunStats({ testRunId }: { testRunId: string }) {
 
   return (
     <div className="flex shrink space-x-2">
-      {failed > 0 && <Pill className={styles.failedPill} value={failed} />}
-      {flakyCount > 0 && <Pill className={styles.flakyPill} value={flakyCount} />}
-      {passed > 0 && <Pill className={styles.successPill} value={passed} />}
+      {failed > 0 && <Pill status="failed" className={styles.failedPill} value={failed} />}
+      {flakyCount > 0 && <Pill status="flaky" className={styles.flakyPill} value={flakyCount} />}
+      {passed > 0 && <Pill status="success" className={styles.successPill} value={passed} />}
     </div>
   );
 }
 
-function Pill({ className, value }: { className: string; value: number }) {
+function Pill({
+  status,
+  className,
+  value,
+}: {
+  status: "failed" | "success" | "flaky";
```

Comment:
We probably don't need `status` and `className` since the latter can be inferred from the former.